src/prettify.js

```diff
@@ -159,17 +159,19 @@ export const prettify = (html, config) => {
   const ignore = validated_config.ignore.length > 0
   trim = validated_config.trim
 
+  const protectionString = config?.protection_string || '_!i-£___£%_';
```

Comment:
Honestly there wasn't a lot of though behind it. It is a random series of characters that would be unlikely to be included in someone's HTML. I also wanted to avoid any "special" characters in HTML to avoid the rest of the functions messing with it (such as `<`, `>`, `&` etc)